jsgui/src/main/webapp/index.html

```diff
@@ -48,6 +51,7 @@
             <div class="nav-collapse menubar-top">
```

Comment:
Believe `<a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">...<a class="brand"...>` above is no longer needed as well.